assets/thanos-ruler/alertmanagers-config-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+data:
+  alertmanagers.yaml: ImFsZXJ0bWFuYWdlcnMiOgotICJodHRwX2NvbmZpZyI6CiAgICAiYmVhcmVyX3Rva2VuX2ZpbGUiOiAiL3Zhci9ydW4vc2VjcmV0cy9rdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3Rva2VuIgogICAgInRsc19jb25maWciOgogICAgICAiY2FfZmlsZSI6ICIvZXRjL3Byb21ldGhldXMvY29uZmlnbWFwcy9zZXJ2aW5nLWNlcnRzLWNhLWJ1bmRsZS9zZXJ2aWNlLWNhLmNydCIKICAgICAgInNlcnZlcl9uYW1lIjogImFsZXJ0bWFuYWdlci1tYWluLm9wZW5zaGlmdC1tb25pdG9yaW5nLnN2YyIKICAic2NoZW1lIjogImh0dHBzIgogICJzdGF0aWNfY29uZmlncyI6CiAgLSAiYWxlcnRtYW5hZ2VyLW1haW4ub3BlbnNoaWZ0LW1vbml0b3Jpbmcuc3ZjOjkwOTQi
```

Comment:
nit: we had some trick have this as plaintext.

Reply:
Did we? No clue about that, but we do have it in plain text in jsonnet. But if you can point me to the trick I can adjust this as well, thanks!

Reply:
Figured out :)